src/raven.js

```diff
@@ -1905,6 +1905,7 @@ Raven.prototype = {
 
     if (supportsFetch()) {
       var fetchOptions = {
+        keepalive: true,
```

Comment:
this means that if someone logs an error in an `unload`, or `beforeunload` handler the request will still hit sentry.